src/mongo_types.ts

```diff
@@ -474,6 +474,38 @@ export class TypedEventEmitter<Events extends EventsDescription> extends EventEm
 /** @public */
 export class CancellationToken extends TypedEventEmitter<{ cancel(): void }> {}
 
+/** @public */
+export type Abortable = {
+  /**
+   * When provided the corresponding `AbortController` can be used to cancel an asynchronous action.
+   *
+   * The driver will convert the abort event into a promise rejection with an error that has the name `'AbortError'`.
```

Comment:
Are there any promise returning APIs that *don't* reject if the controller is aborted?  I think the phrasing here is redundant